ads/_config.js

```diff
@@ -292,6 +292,10 @@ export const adConfig = {
 
   ibillboard: {},
 
+  imedia: {
+    prefetch: 'https://i.imedia.cz/js/im3.js',
```

Comment:
we have a `render-start` API that can help to improve user experience. Please consider implementing [it](https://github.com/ampproject/amphtml/blob/master/ads/README.md#available-apis).
Please let me know if you have any questions.